api/hostdb.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+// ActiveHosts is the struct that pads the response to the hostdb module call
+// "ActiveHosts". The padding is used so that the return value can have an
+// explicit name, which makes adding or removing fields easier in the future.
+type ActiveHosts struct {
+	Entries []modules.HostEntry
+}
```

Comment:
I guess I see the value of it, it just strikes me as needlessly verbose right now. Probably because I can't think of what else you would want to include in that struct. But I suppose if you're going to do it here you should do it for the renter as well.